packages/mui-base/src/Slider/SliderOutput/SliderOutput.tsx

```diff
@@ -0,0 +1,64 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { useComponentRenderer } from '../../utils/useComponentRenderer';
+import { useSliderContext } from '../Root/SliderProvider';
+import { sliderStyleHookMapping } from '../Root/styleHooks';
+import { SliderOutputProps } from './SliderOutput.types';
+import { useSliderOutput } from './useSliderOutput';
+
+function defaultRender(props: React.ComponentPropsWithRef<'output'>) {
+  return <output {...props} />;
+}
+
+const SliderOutput = React.forwardRef(function SliderOutput(
+  props: SliderOutputProps,
+  forwardedRef: React.ForwardedRef<HTMLOutputElement>,
+) {
+  const { render: renderProp, className, ...otherProps } = props;
+
+  const render = renderProp ?? defaultRender;
+
+  const { ownerState, subitems, values } = useSliderContext();
+
+  const { getRootProps } = useSliderOutput({
+    subitems,
+    rootRef: forwardedRef,
+  });
+
+  const { renderElement } = useComponentRenderer({
+    propGetter: getRootProps,
+    render,
+    ownerState,
+    className,
+    ref: forwardedRef,
+    extraProps: {
+      children: values.join(' – '),
```

Comment:
I'd say this format should be customizable.

Reply:
Planned to do this separately! I've opened an issue https://github.com/mui/base-ui/issues/459